components/widgets/menu/menu.jsx

```diff
@@ -58,17 +66,20 @@ export default class Menu extends React.PureComponent {
                 active={this.state.focusTrapped}
                 focusTrapOptions={{
                     clickOutsideDeactivates: true,
+                    initialFocus: this.focusContainer,
```

Comment:
does passing the component directly here work?

Reply:
Nops, if I do:
`initialFocus: this.node.current,` that does not work.